src/lib/errors/sentry.js

```diff
@@ -5,5 +5,22 @@ export default (() => {
     Sentry.init({ dsn: process.env.SENTRY });
   }
 
-  return Sentry;
+  return {
+    ...Sentry,
+    configureMessageScope: message => {
+      Sentry.configureScope(scope => {
+        const { username, id } = message.author;
+        scope.setUser({ username, id });
+
+        if (message.guild) {
+          scope.setExtra('Guild ID', message.guild.id);
+          scope.setExtra('Channel ID', message.channel.id);
+        } else {
+          scope.setExtra('Channel Type', 'DM');
+        }
+
```

Comment:
Add the message content too, and if it contains a file or attachment (and the url to that if it does?)